### ISSUE TITLE
Curl layout instead of Bitmap

### ISSUE THREAD

**Issue by fatahnasrat** [harism/android-pagecurl#127]
how to curl an interactive layout instead of a bitmap? I have a layout where users can add some notes for themselves. so I'm trying to use the Curl Page but it needs to bitmap. if I convert my layout to bitmap and send it to the Curl Page it works but the user cant adds notes anymore with bitmap.

**Comment by kcochibili** [comment]
I don't think this library was designed to do what you're trying to achieve. Maybe when the user clicks the bitmap, take them to the corresponding layout

**Comment by pramod-dexati** [comment]
You can create Bitmap from view and then pass to the Library


public Bitmap loadBitmapFromView(View viewToDrawFrom) {

        boolean wasDrawingCacheEnabled = viewToDrawFrom.isDrawingCacheEnabled();
        if (!wasDrawingCacheEnabled)
            viewToDrawFrom.setDrawingCacheEnabled(true);
        if (width <= 0 || height <= 0) {
            if (viewToDrawFrom.getWidth() <= 0 || viewToDrawFrom.getHeight() <= 0) {
                viewToDrawFrom.measure(View.MeasureSpec.makeMeasureSpec(0, View.MeasureSpec.UNSPECIFIED),
                        View.MeasureSpec.makeMeasureSpec(0, View.MeasureSpec.UNSPECIFIED));
                width = viewToDrawFrom.getMeasuredWidth();
                height = viewToDrawFrom.getMeasuredHeight();
            }
            if (width <= 0 || height <= 0) {
                final Bitmap bmp = viewToDrawFrom.getDrawingCache();
                final Bitmap result = bmp == null ? null : Bitmap.createBitmap(bmp);
                if (!wasDrawingCacheEnabled)
                    viewToDrawFrom.setDrawingCacheEnabled(false);
                return result;
            }
            viewToDrawFrom.layout(0, 0, width, height);
        } else {
            viewToDrawFrom.measure(View.MeasureSpec.makeMeasureSpec(width,
                    View.MeasureSpec.EXACTLY), View.MeasureSpec.makeMeasureSpec(height, View.MeasureSpec.EXACTLY));
            viewToDrawFrom.layout(0, 0, viewToDrawFrom.getMeasuredWidth(), viewToDrawFrom.getMeasuredHeight());
        }
        final Bitmap drawingCache = viewToDrawFrom.getDrawingCache();
        final Bitmap bmp = ThumbnailUtils.extractThumbnail(drawingCache, width, height);
        final Bitmap result = bmp == null || bmp != drawingCache ? bmp : Bitmap.createBitmap(bmp);
        if (!wasDrawingCacheEnabled)
            viewToDrawFrom.setDrawingCacheEnabled(false);
        return result;
    }
